quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/inheritance/solution/TimefoldProcessorBothClassesAnnotatedTest.java

```diff
@@ -26,7 +26,7 @@ class TimefoldProcessorBothClassesAnnotatedTest {
                     .addClasses(TestdataBothAnnotatedExtendedConstraintProvider.class,
                             TestdataBothAnnotatedConstraintProvider.class, TestdataBothAnnotatedExtendedSolution.class,
                             TestdataBothAnnotatedSolution.class, TestdataBothAnnotatedChildEntity.class,
-                            TestdataBothAnnotatedBaseEntity.class))
+                            TestdataEntity.class))
```

Comment:
Isn't this a change of behavior?
The entity wasn't required previously, now it is. Why?